connection_notifier.hh

```diff
@@ -20,19 +20,46 @@
  */
 #pragma once
 
-#include "gms/inet_address.hh"
+#include "db/query_context.hh"
+
+#include <seastar/net/inet_address.hh>
 #include <seastar/core/sstring.hh>
+#include "seastarx.hh"
+
 #include <optional>
 
+namespace db::system_keyspace {
+extern const char *const CLIENTS;
+}
+
 enum class client_type {
     cql = 0,
     thrift,
     alternator,
 };
 
+sstring to_string(client_type ct);
+
+enum class changed_column {
+    username = 0,
+    connection_stage,
+    driver_name,
+    driver_version,
+    hostname,
+    protocol_version,
+};
+
+template <changed_column column> constexpr const char* column_literal = "";
+template <> constexpr const char* column_literal<changed_column::username> = "username";
+template <> constexpr const char* column_literal<changed_column::connection_stage> = "connection_stage";
+template <> constexpr const char* column_literal<changed_column::driver_name> = "driver_name";
+template <> constexpr const char* column_literal<changed_column::driver_version> = "driver_version";
+template <> constexpr const char* column_literal<changed_column::hostname> = "hostname";
+template <> constexpr const char* column_literal<changed_column::protocol_version> = "protocol_version";
+
```

Comment:
Would it make sense to use string_view here?

Reply:
While technically possible (`string_view` has trivial dtor and constexpr methods), I don't see much of an advantage in doing this.
The literals are short, they don't have much in common (so pooling them won't win us much). And `const char*` can be used as template param if we ever decide to build a full-blown compile-time query generator.

Reply:
ok